pkg/proxy/apis/kubeproxyconfig/validation/validation.go

```diff
@@ -238,3 +240,31 @@ func validateIPVSSchedulerMethod(scheduler kubeproxyconfig.IPVSSchedulerMethod,
 	}
 	return allErrs
 }
+
+func validateKubeProxyNodePortAddress(nodePortAddresses []string, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+LOOP:
+	for i := range nodePortAddresses {
+		switch nodePortAddresses[i] {
```

Comment:
Why is this a switch with a single case?  shouldn't that be an `if`?

Reply:
Done